lib/convection/model/template.rb

```diff
@@ -76,6 +82,10 @@ def resource(name, &block)
         resources[name] = r
       end
 
+      def resource_collection(name, &block)
+        resource_collections[name] = Model::Template::ResourceCollection.new(name, self, &block)
+      end
```

Comment:
What does this method do?  Same question for the "not yet implemented" method below?